def.bzl

```diff
@@ -1,15 +1,18 @@
 load(
     "@local_config_cuda//cuda:build_defs.bzl",
     "cuda_default_copts",
-    "if_cuda",
+    _if_cuda = "if_cuda",
 )
 
 load(
     "@local_config_rocm//rocm:build_defs.bzl",
     "rocm_default_copts",
-    "if_rocm",
+    _if_rocm = "if_rocm",
 )
 
+if_rocm = _if_rocm
+if_cuda = _if_cuda
+
```

Comment:
Why change this ? I can't see the effects of refering if_cuda/if_rocm with _if_cuda/_if_rocm

Reply:
To make it exported from defs.bzl for the future changes.